ci/test_common.sh

```diff
@@ -128,7 +128,7 @@ run_py_tests_async() {
   ENABLE_PYTHON_FUTURE=$3
   SKIP=$4
 
-  CMD_LINE="UCXPY_PROGRESS_MODE=${PROGRESS_MODE} UCXPY_ENABLE_DELAYED_SUBMISSION=${ENABLE_DELAYED_SUBMISSION} UCXPY_ENABLE_PYTHON_FUTURE=${ENABLE_PYTHON_FUTURE} timeout 20m python -m pytest -vs python/ucxx/ucxx/_lib_async/tests/ --durations=50"
+  CMD_LINE="UCXPY_PROGRESS_MODE=${PROGRESS_MODE} UCXPY_ENABLE_DELAYED_SUBMISSION=${ENABLE_DELAYED_SUBMISSION} UCXPY_ENABLE_PYTHON_FUTURE=${ENABLE_PYTHON_FUTURE} timeout 20m python -m pytest -vs python/ucxx/ucxx/_lib_async/tests/ --runslow"
```

Comment:
Deliberately dropping the `--durations` flag?

Reply:
Yes, they haven't been adding too much value I think, plus I've found out that you can enable timestamps from GHA or download the raw log with timestamps, for timeouts that is much more useful, for instance [in this failed nightly test](https://github.com/rapidsai/ucxx/actions/runs/11762964521/job/32766417063):

```
Sun, 10 Nov 2024 07:35:10 GMT
python/ucxx/ucxx/_lib_async/tests/test_send_recv_multi.py::test_send_recv_numpy[|u1-8-1] PASSED
Sun, 10 Nov 2024 07:52:49 GMT
python/ucxx/ucxx/_lib_async/tests/test_send_recv_multi.py::test_send_recv_numpy[|u1-8-16] PASSED
Sun, 10 Nov 2024 07:52:49 GMT
Error: Process completed with exit code 124.
```

It's clear `python/ucxx/ucxx/_lib_async/tests/test_send_recv_multi.py::test_send_recv_numpy[|u1-8-1]` got stuck for 17 minutes until the timeout was raised.

We can always reenable if we find more use for it, but I haven't really looked at those numbers in months.